README.md

```diff
@@ -109,6 +142,27 @@ This should be true if only the fields present in the import should be updated,
 ### sourceType
 A prefix for the <code>externalSystemId</code> to be stored in the system. This field is useful for those organizations that has multiple sources of users. With this field the multiple sources can be separated. The source type is appended to the beginning of the <code>externalSystemId</code> with an underscore, e.g. if the user's <code>externalSystemId</code> in the import is somebody012 and the <code>sourceType</code> is test, the user's <code>externalSystemId</code> will be test_somebody012.
 
+### requestPreference
```

Comment:
### requestPreference
Use this attribute to populate the user Request preference. The Request Preference contains following properties:
<code>holdShelf</code> - required field, should always be true;
<code>delivery</code> - required field, could be <code>true</code> or <code>false</code>;
<code>defaultServicePointId</code> - optional, the id of user's default service point
if <code>delivery</code> is <code>true</code> then next properties can be used
<code>fulfillment</code> - required field, can only have <code>Hold shelf</code> or <code>Delivery</code> value;
<code>defaultDeliveryAddressTypeId</code> - optional, the name of user's address type

Reply:
Thank you for the more detailed description!
After reading I got a thought that `holdShelf` and `delivery` fields are redundant. `holdShelf` - is always true and there is no cause for user to specify it. And `delivery` can be populated based on the `fulfillment` field. Just interesting why this fields are needed.